README.md

```diff
@@ -1,180 +1,113 @@
-﻿# Qiskit Metal
-
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4618153.svg?style=popout-square)](https://doi.org/10.5281/zenodo.4618153) [![Release](https://img.shields.io/github/release/Qiskit/qiskit-metal.svg?style=popout-square)](https://github.com/Qiskit/qiskit-metal/releases) [![License](https://img.shields.io/github/license/Qiskit/qiskit-metal.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0) [![](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/Qiskit/qiskit-metal) [![](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/Qiskit/qiskit-metal)
-
-
-**Qiskit Metal** is an open-source framework for designing quantum devices.
-
-
->  Quantum hardware design and analysis
-
-![Welcome to Qiskit Metal!](docs/images/zkm_banner.png 'Welcome to Qiskit Metal')
-
-### Quantum device design project
-We are thrilled to ask you to join this journey to revolutionize quantum devices. This is a first-of-its-kind, open-source project for engineers and scientists to design superconducting quantum devices with ease.
-
-Qiskit Metal is licensed under Apache 2.0. IBM reserves no copyright over outputs of Qiskit Metal.
-
-#### Watch the recorded tutorials
-The streaming will also be recorded and made available [here](https://www.youtube.com/playlist?list=PLOFEBzvs-VvqHl5ZqVmhB_FcSqmLufsjb) for offline review.
-
-#### Take part in the live tutorials and discussion
-Through June 2021 we are offering live tutorials and Q&A. [Sign up](https://airtable.com/shrxQEgKqZCf319F3) to receive an invite to the upcoming sessions.  The streaming will also be recorded and made available for offline review.  Find [here](https://github.com/Qiskit/qiskit-metal/blob/main/README_Tutorials.md) more details on schedule and use the Slack channel to give us feedback and to request the most relevant content to you.
-
-#### Get help: Slack
-Use the slack channel.  Join [qiskit slack](https://ibm.co/joinqiskitslack) and then join the `#metal` channel to communicate with the developers and other participants.  You may also use this channel to inquire about collaborations.
-
-## Documentation
-Full code documentation can be found [here](https://qiskit.org/documentation/metal/)
-
-You can open the page also by executing:
-```python
-import qiskit_metal
-qiskit_metal.open_docs()
+# Qiskit Metal 
+[![License](https://img.shields.io/github/license/Qiskit/qiskit-metal.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)<!--- long-description-skip-begin -->[![Release](https://img.shields.io/github/release/Qiskit/qiskit-metal.svg?style=popout-square)](https://github.com/Qiskit/qiskit-metal/releases)<!--- long-description-skip-begin -->[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4618153.svg)](https://doi.org/10.5281/zenodo.4618153)
```

Comment:
Well, I'm a little bit sad to see the open-source and awesome icons go, but I suppose that they can be seen as more clutter than cute.

Reply:
I can add them back! Do you want me to?

Reply:
Consider the consequences of the two choices, and what each might lead to in terms of experience and subtext, then make a call. I'm good either way￼￼